benchmarks/experiment_runner.py

```diff
@@ -44,97 +44,113 @@ def __init__(self, args):
     self.output_file = os.path.join(self.output_dir, self._args.output_basename)
 
   def run(self):
-    if self._args.experiment_config and self._args.model_config:
-      if self._args.dry_run:
-        logger.warning(f"Dry run with {[sys.executable] + sys.argv}")
-        return
-      experiment_config = json.loads(self._args.experiment_config)
-      model_config = json.loads(self._args.model_config)
-      self.run_single_experiment(experiment_config, model_config)
+    is_entry_process = self._args.experiment_config is None and self._args.model_config is None
```

Comment:
by "entry_process", do you mean the main process?

Reply:
Yes, changed the name.